mpp/include/mpp_domains_define.inc

```diff
@@ -1347,6 +1349,9 @@ end subroutine check_message_size
        end if
     end do
 
+    allocate(domain%tile_id_all(num_tile))
```

Comment:
This is a comment about code not touched:  The `tile_id_local` allocatable array is not deallocated in this routine.  While most compilers will, possibly, automatically deallocate it is much safer to add a deallocate in the routine.  I'm not going to have this hold up the merge, but it is something that should be addressed.